ecommerce/templates/oscar/checkout/payment_pending.html

```diff
@@ -0,0 +1,27 @@
+{% extends 'edx/base.html' %}
+{% load i18n %}
+{% load staticfiles %}
+
+{% block title %}
+    {% trans "Pending Payment" %}
+{% endblock %}
+
+{% block navbar %}
+    {% include 'edx/partials/_student_navbar.html' %}
+{% endblock %}
+
+{% block content %}
+    <div id="error-message">
+        <div class="container">
+            <div class="depth depth-2 message-error-content">
+                <h1>{% trans "Pending Payment" %}</h1>
+                {% if message %}
+                    <p>{% trans message %} </p>
+                {% else %}
+                    <p>{% trans "Your payment with Paypal is pending. Until it is approved, you will not get access to the course. We will notify you when it is approved and provide you access at that time." %} </p>
```

Comment:
@andrey-canon Could you set Paypal as a variable taken from the payment processor name ?